wp-seo.php

```diff
@@ -1,5 +1,10 @@
 <?php
-/*
+/**
+ * @package    WPSEO
+ * @subpackage Main
+ */
+
+/**
```

Comment:
Good point, I wonder why I hadn't got to just looking that up at the time. Pushed a commit with docblock formatting.